packages/loopring_v3/contracts/test/AmmPool.sol

```diff
@@ -271,12 +271,16 @@ contract AmmPool is IBlockReceiver {
         return shutdownTimestamp == 0;
     }
 
-
     /// @dev Joins the pool using on-chain funds.
-    /// @param poolAmountOut The number of liquidity tokens that need to be minted for this join.
+    /// @param minLiquidityTokenToMint The minimul number of liquidity tokens that is expected
+    ///                                to be minted for this join. If this requirement cannot be
+    ///                                met, this join should fail.
```

Comment:
Will you change the minting/burning logic? Currently there is no min or max amount of liquidity tokens to mint/burn, it will always mint/burn exactly the amount of liquidity tokens specified in the request. The amounts that can differ is the amounts deposited/withdrawn (which have use min/max prefix).

Reply:
I am still digesting the code, and don't have intention to change any logic.